docs/user/security/index.asciidoc

```diff
@@ -47,4 +47,5 @@ include::authorization/kibana-privileges.asciidoc[]
 include::api-keys/index.asciidoc[]
 include::encryption-keys/index.asciidoc[]
 include::role-mappings/index.asciidoc[]
+include::tutorials/how-to-secure-access-to-kibana.asciidoc[]
 include::rbac_tutorial.asciidoc[]
```

Comment:
@gchaps there's an existing RBAC tutorial here which has some overlap. I only remembered it when I went to link this one in 🙂.

I rather like the format of the existing tutorial, but it makes use of more screenshots, which I think(?) we're trying to avoid. The new tutorial has a few screenshots, but I can rewrite it to be in the style of the existing tutorial if you think that'd be helpful.